charts/common/tests/hpa_test.yaml

```diff
@@ -0,0 +1,144 @@
+suite: test runtime bundle hpa
+templates:
+  - hpa.yaml
+tests:
+  - it: should render nothing if not enabled
+    set:
+      enabled: true
+      hpa:
+        enabled: false
+    asserts:
+      - hasDocuments:
+          count: 0
+  - it: should render nothing if hpa.enabled=false
+    set:
+      enabled: true
+      hpa:
+        enabled: false
+    asserts:
+      - hasDocuments:
+          count: 0
+  - it: should render minReplicas
```

Comment:
Suggestion: `should render minReplicas and maxReplicas`.

Reply:
Done.